takwimu/templates/takwimu/_includes/profile/country_section.html

```diff
@@ -4,12 +4,12 @@
     <div class="row">
       <div class="col-4">
         <img class=" border-light mr-2"
-             src={% with 'img/maps/'|add:country|lower|add:'.svg' as image_static %}
+             src={% with 'img/maps/'|add:country_slug|lower|add:'.svg' as image_static %}
```

Comment:
`country_slag` should be already in lower

Reply:
@kilemensi let me update